src/components/FiltersToggleBtn/index.js

```diff
@@ -1,36 +1,48 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
```

Comment:
Only learned this yesterday, you probably know it, but I didn't so I'll mention it here (so I won't forget), when possible use `import NamedThing from "lib/path/NamedThing` instead of `import { NamedThing } from "lib"`.

`import { NamedThing } from "lib"` will pull in the entire lib, while the other one won't. (or so I've been told)

Reply:
normally all this gets tree shaked  with webpack but sometimes it doesn't like in lodash. So yes and no. ;) we should install the bundler analyser to check from time to time